package.json

```diff
@@ -4,14 +4,16 @@
     "packages/*"
   ],
   "scripts": {
+    "prebuild": "yarn build:ts",
     "build": "node ./scripts/build.js",
-    "build-clean": "rm -rf ./packages/*/build",
+    "build:ts": "node ./scripts/buildTs.js",
+    "build-clean": "rm -rf ./packages/*/build ./packages/*/tsconfig.tsbuildinfo",
     "watch": "node ./scripts/watch.js",
     "test": "jest",
     "test:ci:unit": "jest packages --ci --coverage",
     "test:ci:e2e": "jest e2e --ci -i",
+    "lint": "eslint --ext .js,.ts . --cache --report-unused-disable-directives",
```

Comment:
Do we need to specify extensions manually even if we use ESLint config?

Reply:
For Typescript, yes.